numpy/_core/tests/test_einsum.py

```diff
@@ -73,6 +73,61 @@ def test_einsum_errors(self, do_opt, einsum_fn):
             b = np.ones((3, 4, 5))
             einsum_fn('aabcb,abc', a, b)
 
+    def test_einsum_sorting_behavior(self):
+        # Case 1: 26 dimensions (all lowercase indices)
+        n1 = 26
+        x1 = np.random.random((1,) * n1)
+        path1 = np.einsum_path(x1, range(n1))[1]  # Get einsum path details
+        output_indices1 = path1.split("->")[-1].strip()  # Extract output indices
+        # Assert indices are only lowercase letters and sorted correctly
```

Comment:
The test actually checks for upper case. This is a behavioral change, but I think it is OK.

```suggestion
        # Assert indices are only uppercase letters and sorted correctly
```